### DISCUSSION TITLE
YOU ALL ARE AWESOME

### DISCUSSION THREAD

**Discussion by cleanlivin92** [Cyfrin/foundry-full-course-cu#103] [General]
Hello All! Been in crypto since 2016. I run a bitcoin mining farm in Dubai. I have some ideas for the blockchain world and am here to gain the knowledge to be able to speak about this deeply with developers and be able to start working on my idea. Learning solidity and being able to communicate with computers and making smart contracts is really something. You all rock!

**Comment by cromewar** [comment]
Hello there @cleanlivin92 
Happy to have you on the community, you can work on the idea trough a hackathon and join more developers to make it real. Good luck!

**Reply by cleanlivin92** [reply]
Thanks for the feedback! Know of any hackathons happening in lisbon or europe sometime soon?

**Reply by alymurtazamemon** [reply]
@cleanlivin92 Here you can check [ETHGlobal Events](https://ethglobal.com/events)
